identifier/index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head data-kopano-build="%VITE_KOPANO_BUILD%">
     <meta charset="utf-8">
+    <meta http-equiv="Content-Security-Policy">
```

Comment:
So we are basically adding the csp headers from the csphashes of legacy plugin. This will automatically add all the csp headers dynamically to the meta tag. The reason why is its empty , is because it is the src html which when build, the cspPlugin will update the csp hashes in the meta tag.

 htmlContent = htmlContent.replace(
        /<meta http-equiv="Content-Security-Policy">/,
        `<meta http-equiv="Content-Security-Policy" content="${cspHeader}">`
      );
      
Please check the vite config here.

Reply:
This does not work, like I said lico sends csp headers via response header. You can use the existing csp nonce to whitelist certain inline scripts.

Reply:
I have updated with the script tag itself with nonce there. and the cspPlugin and api.go has also been updated.